tools/scripts/check-nrdb.js

```diff
@@ -14,23 +14,29 @@ if (!version) {
   process.exit(1)
 }
 
+const urls = [
+  'https://staging-api.newrelic.com/v2/js_agent_loaders/version.json',
+  'https://api.newrelic.com/v2/js_agent_loaders/version.json',
+  'https://api.eu.newrelic.com/v2/js_agent_loaders/version.json'
+]
+
 var opts = {
-  uri: 'https://api.newrelic.com/v2/js_agent_loaders/version.json',
   method: 'GET',
   gzip: true,
   qs: { loader_version: `nr-loader-spa-${version}.min.js` }
```

Comment:
only spa? x.x versions, and polyfills too, or no?

Reply:
i think we leave this as a basic smell check for now